styles/style.css

```diff
@@ -951,12 +669,8 @@ h1 a {
   background-color: #ffe7dd;
   border-radius: 3px;
   -webkit-animation: tv-flash 0.3s linear 0s 1;
-  -moz-animation: tv-flash 0.3s linear 0s 1;
-  -ms-animation: tv-flash 0.3s linear 0s 1;
   animation: tv-flash 0.3s linear 0s 1;
   -webkit-animation-fill-mode: both;
-  -moz-animation-fill-mode: both;
-  -ms-animation-fill-mode: both;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The property -moz-animation is compatible with -webkit-animation and should be included as well.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1888804921/issues/source?bid=3085146&fileBranchId=3088807#l765)